### ISSUE TITLE
type_annotate_public_apis triggers on private typedef

### ISSUE THREAD

**Issue by pylaligand** [dart-lang/linter#216]
```
typedef _PrivateMethod(int value);
```

This generates a lint warning even though it's supposed to be private.

**Comment by pq** [comment]
Good catch!

Fix in flight: https://codereview.chromium.org/1896893002/

**Comment by pq** [comment]
Fixed w/ https://github.com/dart-lang/linter/commit/d66985aaf9da193a98c214185de2886369e3237c.
